k8s/xdmod-build/bc-xdmod-dev.yaml

```diff
@@ -8,7 +8,6 @@ spec:
     type: Git
     git:
       uri: https://github.com/CCI-MOC/xdmod-cntr.git
-      ref: XdmodK8sDeployment
```

Comment:
Should this be `ref: main` instead so that it's clear what branch it's using by default (and similarly for the other buildconfig)?

Reply:
It should be eliminated as it should default to the main branch.